test/common/router/config_impl_speed_test.cc

```diff
@@ -27,19 +27,21 @@ using testing::ReturnRef;
  * Generates a request with the path:
  * - /shelves/shelf_x/route_x
  */
-static Http::TestRequestHeaderMapImpl genRequestHeaders(int route_num) {
+static Http::TestRequestHeaderMapImpl genRequestHeaders(int route_num,
+                                      std::string req_path_header) {
   return Http::TestRequestHeaderMapImpl{
       {":authority", "www.google.com"},
       {":method", "GET"},
-      {":path", absl::StrCat("/shelves/shelf_", route_num, "/route_", route_num)},
+      {":path", absl::StrCat(req_path_header, route_num, "/route_", route_num)},
       {"x-forwarded-proto", "http"}};
 }
 
 /**
  * Generates the route config for the type of matcher being tested.
  */
 static RouteConfiguration genRouteConfig(benchmark::State& state,
-                                         RouteMatch::PathSpecifierCase match_type) {
+                                         RouteMatch::PathSpecifierCase match_type,
+                                         std::string path_rule_base) {
```

Comment:
`config_path_prefix`? `rule_path_prefix`?

Reply:
@yanjunxiang-google suggest switching to terminology fully consistent with that used in `RouteConfiguration` in xDS, i.e. there is no such thing as "path rule" there, rather than our internal terminology.